web-app/deployment/docker-compose.prod.yaml

```diff
@@ -0,0 +1,42 @@
+version: "3"
+services:
+
+  nestjs:
+    container_name: nestjs
```

Comment:
Ideally you want to use a already built image in production instead of building from a Dockerfile.
The Docker image is called a deployment artifact, and can (ideally again) be used on any environment by customizing the env vars.

Reply:
I'm not sure to see how i could mix this approach with the one suggested by Clement(with a multistage docker image). Could we have a talk about it ?